samples/test/filters.js

```diff
@@ -19,12 +19,14 @@ const snapshot = require('snap-shot-it');
 const {assert} = require('chai');
 const {describe, it, before, after} = require('mocha');
 const cp = require('child_process');
-const {obtainTestInstance} = require('./util');
+const {obtainTestInstance, createTestInstance} = require('./util');
 
 const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
 const runId = uuid.v4().split('-')[0];
 const TABLE_ID = `mobile-time-series-${runId}`;
 
+createTestInstance();
```

Comment:
This will create an `unhandledRejection`, so I think we shouldn't do this in the top level.